src/server/multi_svr.c

```diff
@@ -40,50 +40,77 @@
 /**
  *
  * @brief
- *		all the functions to handle peer server in case of multi-server.
+ *		Functions associated with peersvr structure
  *
  */
 
-#include	<netdb.h>
-#include	<arpa/inet.h>
-#include	"pbs_nodes.h"
-#include	"pbs_error.h"
-#include	"server.h"
-#include	"batch_request.h"
-#include	"svrfunc.h"
-#include	"pbs_nodes.h"
-#include	"tpp.h"
+#include "avltree.h"
+#include "batch_request.h"
+#include "pbs_error.h"
+#include "pbs_nodes.h"
+#include "server.h"
+#include "svrfunc.h"
+#include "tpp.h"
+#include <arpa/inet.h>
+#include <netdb.h>
 
-typedef struct peersvr_list {
-	struct peersvr_list *next;
-	svrinfo_t *psvr;
-} peersvr_list_t;
-
-static struct peersvr_list *peersvrl;
+pbs_list_head peersvrl;
+static void *alien_node_idx;
 
 /**
  * @brief
  *	Get the peer server structure corresponding to the addr
  *
  * @param[in]	addr	- addr contains ip and port
  *
- * @return	svrinfo_t
+ * @return	server_t
  * @retval	NULL	- Could not find peer server corresponing to the addr
  * @retval	!NULL	- svrinfo structure
  */
 void *
 get_peersvr(struct sockaddr_in *addr)
 {
-	svrinfo_t *psvr;
+	server_t *psvr;
 
 	psvr = tfind2(ntohl(addr->sin_addr.s_addr),
-			ntohs(addr->sin_port), &ipaddrs);
+		      ntohs(addr->sin_port), &ipaddrs);
 	if (psvr && psvr->mi_rmport == psvr->mi_port)
 		return psvr;
 
 	return NULL;
 }
 
+/**
+ * @brief Get the peersvr from svrid object
+ * 
+ * @param[in] sv_id - server instance id
+ * 
+ * @return void*
+ * @retval !NULL - success; peer svr object
+ * @retval NULL - failure
+ */
+void *
+get_peersvr_from_svrid(char *sv_id)
+{
+	psi_t psi;
+	server_t *psvr;
+
+	if (!sv_id)
+		return NULL;
```

Comment:
one indent more?

Reply:
the "if" has been indented with one tab and "return" is indented with two.